Libraries/Fetch/fetch.js

```diff
@@ -352,7 +352,7 @@ var self = {};
       var xhr = new XMLHttpRequest()
 
       function responseURL() {
-        if ('responseURL' in xhr) {
+        if ('responseURL' in xhr && xhr.responseURL) {
```

Comment:
Instead of changing this in fetch.js (which is upstream) could we delete `responseURL` if it is null? I'm not 100% sure what this means in this context as I'm guessing it will never be null in our case?

The spec says this:
>    The responseURL attribute must return the empty string if response's url is null and its serialization with the exclude fragment flag set otherwise.

I'd be ok with just deleting the property if it's null. Seems reasonable?